test/web-platform-tests/to-run.yaml

```diff
@@ -103,6 +103,46 @@ window-screen-width.html: [fail, Test not applicable - no output device]
 
 ---
 
+DIR: custom-elements
+
+CustomElementRegistry.html: [fail, webidl2js doesn't deal well with tests using Proxies to verify properties access]
+Document-createElement.html: [fail, :defined is not defined and throws]
+HTMLElement-attachInternals.html: [fail, Not implemented]
+HTMLElement-constructor.html: [fail, webidl2js doesn't deal well with tests using Proxies to verify properties access]
+adopted-callback.html: [fail, "Failing test due to https://github.com/whatwg/dom/issues/813"]
```

Comment:
It seems like there's been some progress on the spec issue here; is this still right?

Reply:
We have [multiple](https://github.com/jsdom/jsdom/blob/master/test/web-platform-tests/to-run.yaml#L161) [tests](https://github.com/jsdom/jsdom/blob/master/test/web-platform-tests/to-run.yaml#L822) that have been disabled because of this issue. I would be happy to work on this in a different PR.